tests/context.py

```
@@ -1003,7 +972,7 @@ def _should_get_package_version_WHEN_valid(self):
 	def test_absolute_truth_and_meaning(self):
 		"""Test case 0: Insanitty Test."""
 		assert True
-		self.assertTrue(True, "Insanitty Test Failed")
+		self.assertTrue(True, "Insanitty Test Failed")  # skipcq: PYL-W1503
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)